setup.py

```diff
@@ -0,0 +1,53 @@
+from setuptools import setup
+from codecs import open
+from os import path
+
+here = path.abspath(path.dirname(__file__))
+
+# Get the long description from the relevant file
+with open(path.join(here, 'README.md'), encoding='utf-8') as f:
+    long_description = f.read()
+
+setup(
```

Comment:
Not much to say here, but reading this made me do some reading about how python packages work which is definitely helpful for me in the future 📦 .